# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/ClientWebGLContext.h
# Commit: bc1b6e82d67f
# Full Hash: bc1b6e82d67fd6018b946c384efa6be9a5a5af16
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2021-12-10 21:58:52
# Regressor Bug: 1736177
# File Overlap Count: 2
# Description:
#   Bug 1736177 - Part 11. Plumb in OffscreenCanvasDisplayHelper. r=jgilbert
#   
#   This patch integrates OffscreenCanvasDisplayHelper with
#   HTMLCanvasElement, OffscreenCanvas and nsDisplayCanvas to allow
#   asynchronous display of an OffscreenCanvas.
# ==============================================================================

diff -r 7b946576bafa -r bc1b6e82d67f dom/canvas/ClientWebGLContext.h
--- a/dom/canvas/ClientWebGLContext.h	Fri Dec 10 02:57:55 2021 +0000
+++ b/dom/canvas/ClientWebGLContext.h	Fri Dec 10 02:57:55 2021 +0000
@@ -771,6 +771,8 @@
   bool CreateHostContext(const uvec2& requestedSize);
   void ThrowEvent_WebGLContextCreationError(const std::string&) const;
 
+  void UpdateCanvasParameters();
+
  public:
   void MarkCanvasDirty();
 